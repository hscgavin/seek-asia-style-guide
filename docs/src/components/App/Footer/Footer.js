import styles from './Footer.less';
import React from 'react';
import { PageBlock, Section } from 'seek-asia-style-guide/react';
import GithubIcon from './GithubIcon/GithubIcon';

export default function Footer() {
  return (
    <PageBlock>
      <Section>
        <div className={styles.root}>
          <a
            className={styles.link}
            href="https://github.com/seek-oss/seek-asia-style-guide"
            title="View project on Github">
            <GithubIcon svgClassName={styles.github} />
          </a>
        </div>
      </Section>
    </PageBlock>
  );
}
