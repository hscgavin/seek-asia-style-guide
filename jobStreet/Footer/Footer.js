import React from 'react';
import styles from './footer.less';
import PropTypes from 'prop-types';

import FooterLinks from './components/FooterLinks/FooterLinks';
import localization from '../localization';

const Footer = ({ language, country }) => {
  const year = new Date().getFullYear();
  const messages = localization[`${language}-${country}`];

  return (
    <footer className={styles.container}>
      <FooterLinks messages={messages} />
      <p className={styles.copyright}>
        {messages['footer.copyright'].replace('{year}', year)}
      </p>
    </footer>
  );
};

Footer.propTypes = {
  language: PropTypes.string.isRequired,
  country: PropTypes.string.isRequired
};

export default Footer;
