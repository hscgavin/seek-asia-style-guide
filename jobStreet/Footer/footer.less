.container {
    margin: 0 auto;
    padding: 0 20px;
    @media(min-width: 769px) {
        padding: 0 50px;
    }
    margin-bottom: 20px;

    font-size: 14px;
    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
}
.copyright {
    color: #757575;
    font-size: 12px;
    margin-top: 10px;
    text-align: center;

    @media(min-width: 769px) {
        text-align: right;
    }
}
