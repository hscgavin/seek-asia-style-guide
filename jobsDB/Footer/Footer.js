import React from 'react';
import styles from './footer.less';
import PropTypes from 'prop-types';

import FooterLinks from './components/FooterLinks/FooterLinks';
import localization from './localization';

const Footer = ({ language, country }) => {
    const year = new Date().getFullYear();
    const messages = localization[`${language}-${country}`];

    return (
        <footer className={styles.container}>
          <div className={styles.wrapper}>
            <div className={styles.pullRight}><FooterLinks messages={ messages } /></div>
            <div className={styles.pullLeft}>
              <p className={styles.copyright}>{ messages['footer.copyright'].replace('{year}', year) }<span className={styles.fullversion}>{ messages['footer.rightReserved']}</span></p>
            </div>
          </div>
        </footer>
    );
};

Footer.PropTypes = {
  language: PropTypes.string.isRequired,
  country: PropTypes.string.isRequired
};

export default Footer;
