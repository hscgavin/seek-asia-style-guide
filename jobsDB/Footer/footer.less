@import (reference) "~seek-asia-style-guide/theme";

.container {
  margin: 0 auto;
  padding: 0 20px;
  width: 100%;
  background-color: #0f2c55;
  font-family:
    "Helvetica Neue",
    Helvetica,
    Arial,
    sans-serif;
  @media @mobile {
    padding: 0 8px;
    margin-bottom: @grid-base * 14;
  }
}

.wrapper {
  max-width: 970px;
  margin: 0 auto;
}

.copyright {
  color: @saWhite;
  font-size: 0.75rem;
  text-align: center;
}

.pullLeft {
  float: left;
  padding: 5px 0;
  @media @desktop {
    padding: 12px 0;
  }
}

.pullRight {
  float: right;
  padding: 5px 0;
  @media @desktop {
    padding: 12px 0;
  }
}

.fullversion {
  @media @mobile {
    display: none;
  }
}
