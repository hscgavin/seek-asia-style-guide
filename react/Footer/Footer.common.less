@import (reference) "~seek-asia-style-guide/theme";

@footer-vertical: ~"and (max-width: " (@responsive-breakpoint - 1) ~")";
@footer-horizontal: ~"and (min-width: " @responsive-breakpoint ~")";
@footer-classic: ~"and (min-width: 1025px)";

.footerText() {
  .rawText();
  line-height: (@grid-row-height * 4);
  height: (@grid-row-height * 4);
  @media only screen @footer-vertical {
    .touchableText(@conversational-type-scale);
  }
}

.footerLink() {
  .link();
  display: inline-block;
  color: @sk-link-on-mid-gray-light;
  @media only screen @footer-vertical {
    .touchableText(@conversational-type-scale);
  }
  @media only screen @footer-horizontal {
    .footerText();
  }
}
