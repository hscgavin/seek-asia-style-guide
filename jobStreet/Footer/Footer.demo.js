import Footer from './Footer';

export default {
  route: '/footer',
  title: 'JobStreet Footer',
  component: Footer,
  initialProps: {
    language: 'en',
    country: 'my'
  },
  options: [
    {
      label: 'Language',
      type: 'radio',
      states: [
        {
          label: 'English',
          transformProps: props => ({
            ...props,
            language: 'en'
          })
        },
        {
          label: 'Bahasa',
          transformProps: props => ({
            ...props,
            language: 'id'
          })
        },
        {
          label: 'Vietnamese',
          transformProps: props => ({
            ...props,
            language: 'vi'
          })
        }
      ]
    },
    {
      label: 'Country',
      type: 'radio',
      states: [
        {
          label: 'Malaysia',
          transformProps: props => ({
            ...props,
            country: 'my'
          })
        },
        {
          label: 'Singapore',
          transformProps: props => ({
            ...props,
            country: 'sg'
          })
        },
        {
          label: 'Indonesia',
          transformProps: props => ({
            ...props,
            country: 'id'
          })
        },
        {
          label: 'Phillipines',
          transformProps: props => ({
            ...props,
            country: 'ph'
          })
        },
        {
          label: 'Vietnam',
          transformProps: props => ({
            ...props,
            country: 'vn'
          })
        }
      ]
    }
  ]
};
