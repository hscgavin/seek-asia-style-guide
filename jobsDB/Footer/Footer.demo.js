import Footer from './Footer';

export default {
  route: '/footer',
  tenantPath: 'jobsDB',
  title: 'Footer',
  component: Footer,
  initialProps: {
    language: 'en',
    country: 'hk'
  },
  options: [
    {
      label: 'Language',
      type: 'radio',
      states: [
        {
          label: 'English',
          transformProps: props => ({
            ...props,
            language: 'en'
          })
        },
        {
          label: 'Bahasa',
          transformProps: props => ({
            ...props,
            language: 'id'
          })
        },
        {
          label: 'Thai',
          transformProps: props => ({
            ...props,
            language: 'th'
          })
        }
      ]
    },
    {
      label: 'Country',
      type: 'radio',
      states: [
        {
          label: 'Hong Kong',
          transformProps: props => ({
            ...props,
            country: 'hk'
          })
        },
        {
          label: 'Singapore',
          transformProps: props => ({
            ...props,
            country: 'sg'
          })
        },
        {
          label: 'Indonesia',
          transformProps: props => ({
            ...props,
            country: 'id'
          })
        },
        {
          label: 'Thailand',
          transformProps: props => ({
            ...props,
            country: 'th'
          })
        }
      ]
    }
  ]
};
