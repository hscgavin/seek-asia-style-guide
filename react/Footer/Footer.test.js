import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

const renderFooter = props => renderer.create(<Footer {...props} />);

describe('Footer:', () => {
  it('should render with locale of AU', () => {
    expect(renderFooter({ locale: 'AU' }).toJSON()).toMatchSnapshot();
  });

  it('should render with locale of NZ', () => {
    expect(renderFooter({ locale: 'NZ' }).toJSON()).toMatchSnapshot();
  });

  it('should render when authenticated', () => {
    expect(renderFooter({ authenticationStatus: 'authenticated' }).toJSON()).toMatchSnapshot();
  });

  it('should render when unauthenticated', () => {
    expect(renderFooter({ authenticationStatus: 'unauthenticated' }).toJSON()).toMatchSnapshot();
  });

  it('should render when authentication is pending', () => {
    expect(renderFooter({ authenticationStatus: 'pending' }).toJSON()).toMatchSnapshot();
  });
});
