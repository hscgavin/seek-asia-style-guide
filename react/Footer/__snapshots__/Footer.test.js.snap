// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer: should render with locale of AU 1`] = `
<footer
  aria-labelledby="FooterHeading"
  className="root"
  role="contentinfo"
>
  <Hidden
    className=""
    desktop={false}
    mobile={false}
    print={true}
    screen={false}
  >
    <PageBlock>
      <section>
        <ScreenReaderOnly>
          <h1
            id="FooterHeading"
          >
            Footer
          </h1>
        </ScreenReaderOnly>
        <div
          className="content"
        >
          <div
            className="columns"
          >
            <FooterNav
              label="Tools"
              secondary={false}
            >
              <FooterLink
                analytics="toolbar:jobs"
                href="/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Job search
              </FooterLink>
              <FooterLink
                analytics="toolbar:profile"
                href="/profile/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Profile
              </FooterLink>
              <FooterLink
                analytics="toolbar:my+activity"
                href="/myactivity#favourite"
                linkRenderer={[Function]}
                secondary={false}
              >
                Saved searches
              </FooterLink>
              <FooterLink
                analytics="toolbar:favourite"
                href="/my-activity/saved-jobs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Saved jobs
              </FooterLink>
              <FooterLink
                analytics="toolbar:applied"
                href="/my-activity/applied-jobs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Applied jobs
              </FooterLink>
              <FooterLink
                analytics="toolbar:advice+tips"
                href="/career-advice/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Advice & tips
              </FooterLink>
              <FooterLink
                analytics="toolbar:companies"
                href="/companies/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Company reviews
              </FooterLink>
              <ToggleContainer
                label="SEEK sites"
                name="PartnerSitesToggle"
                secondary={false}
              >
                <FooterLink
                  analytics="toolbar:post+a+job+ad"
                  href="https://talent.seek.com.au/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Employer site
                </FooterLink>
                <FooterLink
                  analytics="toolbar:nz+homepage"
                  href="https://www.seek.co.nz/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  SEEK NZ
                </FooterLink>
                <FooterLink
                  analytics="toolbar:courses"
                  href="https://www.seeklearning.com.au/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Courses
                </FooterLink>
                <FooterLink
                  analytics="toolbar:business+for+sale"
                  href="https://www.seekbusiness.com.au/?tracking=sk:main:au:nav:bus"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Business for sale
                </FooterLink>
                <FooterLink
                  analytics="toolbar:volunteering"
                  href="https://www.volunteer.com.au/?tracking=SKMAU:main+header"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Volunteering
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Company"
              secondary={true}
            >
              <FooterLink
                analytics="toolbar:about+seek"
                href="/investor/about-us"
                linkRenderer={[Function]}
                secondary={true}
              >
                About SEEK
              </FooterLink>
              <FooterLink
                analytics="toolbar:media"
                href="/investor/news-media"
                linkRenderer={[Function]}
                secondary={true}
              >
                Media
              </FooterLink>
              <FooterLink
                analytics="toolbar:investor+centre"
                href="https://ir.seek.com.au"
                linkRenderer={[Function]}
                secondary={true}
              >
                Investor centre
              </FooterLink>
              <ToggleContainer
                label="International partners"
                name="InternationalPartnersToggle"
                secondary={true}
              >
                <FooterLink
                  analytics="toolbar:babajob"
                  href="http://www.babajob.com"
                  linkRenderer={[Function]}
                  partner="India"
                  secondary={false}
                >
                  BabaJob
                </FooterLink>
                <FooterLink
                  analytics="toolbar:bdjobs"
                  href="http://www.bdjobs.com"
                  linkRenderer={[Function]}
                  partner="Bangladesh"
                  secondary={false}
                >
                  Bdjobs
                </FooterLink>
                <FooterLink
                  analytics="toolbar:brighter+monday"
                  href="https://www.brightermonday.com"
                  linkRenderer={[Function]}
                  partner="E Africa"
                  secondary={false}
                >
                  Brighter Monday
                </FooterLink>
                <FooterLink
                  analytics="toolbar:catho"
                  href="http://www.catho.com.br"
                  linkRenderer={[Function]}
                  partner="Brazil"
                  secondary={false}
                >
                  Catho
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobberman"
                  href="http://www.jobberman.com"
                  linkRenderer={[Function]}
                  partner="W Africa"
                  secondary={false}
                >
                  Jobberman
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobsdb"
                  href="http://www.jobsdb.com"
                  linkRenderer={[Function]}
                  partner="SE Asia"
                  secondary={false}
                >
                  jobsDB
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobstreet"
                  href="http://www.jobstreet.com"
                  linkRenderer={[Function]}
                  partner="SE Asia"
                  secondary={false}
                >
                  JobStreet
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jora+int"
                  href="https://www.jora.com"
                  linkRenderer={[Function]}
                  partner="Worldwide"
                  secondary={false}
                >
                  Jora
                </FooterLink>
                <FooterLink
                  analytics="toolbar:manager"
                  href="http://www.manager.com.br"
                  linkRenderer={[Function]}
                  partner="Brazil"
                  secondary={false}
                >
                  Manager
                </FooterLink>
                <FooterLink
                  analytics="toolbar:occ+mundial"
                  href="https://www.occ.com.mx"
                  linkRenderer={[Function]}
                  partner="Mexico"
                  secondary={false}
                >
                  OCC Mundial
                </FooterLink>
                <FooterLink
                  analytics="toolbar:workana"
                  href="https://www.workana.com"
                  linkRenderer={[Function]}
                  partner="Latin America"
                  secondary={false}
                >
                  Workana
                </FooterLink>
                <FooterLink
                  analytics="toolbar:zhaopin"
                  href="http://www.zhaopin.com"
                  linkRenderer={[Function]}
                  partner="China"
                  secondary={false}
                >
                  Zhaopin
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jora+au"
                  href="https://au.jora.com"
                  linkRenderer={[Function]}
                  partner="Australia"
                  secondary={false}
                >
                  Jora
                </FooterLink>
              </ToggleContainer>
              <ToggleContainer
                data-automation="partner-services-toggle"
                label="Partner services"
                name="PartnerServicesToggle"
                secondary={true}
              >
                <FooterLink
                  analytics="toolbar:jobadder"
                  href="https://jobadder.com/au"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  JobAdder
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jora+local"
                  data-automation="jora-local-link"
                  href="http://www.joralocal.com.au"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Jora Local
                </FooterLink>
                <FooterLink
                  analytics="toolbar:realestate+au"
                  href="https://www.realestate.com.au/buy"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  realestate.com.au
                </FooterLink>
                <FooterLink
                  analytics="toolbar:sidekicker"
                  href="https://www.sidekicker.com.au/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Sidekicker
                </FooterLink>
                <FooterLink
                  analytics="toolbar:ximble"
                  href="https://www.ximble.com"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Ximble
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Connect"
              secondary={false}
            >
              <FooterLink
                analytics="toolbar:contact+us"
                href="/contact-us"
                linkRenderer={[Function]}
                secondary={false}
              >
                Contact us / FAQs
              </FooterLink>
              <FooterLink
                analytics="toolbar:work+for+seek"
                href="/work-for-seek"
                linkRenderer={[Function]}
                secondary={true}
              >
                Work for SEEK
              </FooterLink>
              <FooterLink
                analytics="toolbar:product+blog"
                href="http://productblog.seek.com.au"
                linkRenderer={[Function]}
                secondary={true}
              >
                Product & Tech Blog
              </FooterLink>
              <ToggleContainer
                label="Social"
                name="SocialToggle"
                secondary={false}
              >
                <FooterLink
                  analytics="toolbar:facebook"
                  href="https://www.facebook.com/SEEK/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Facebook
                </FooterLink>
                <FooterLink
                  analytics="toolbar:instagram"
                  href="https://www.instagram.com/seekau/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Instagram
                </FooterLink>
                <FooterLink
                  analytics="toolbar:twitter"
                  href="https://twitter.com/seekjobs"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Twitter
                </FooterLink>
                <FooterLink
                  analytics="toolbar:youtube"
                  href="https://www.youtube.com/user/SEEKJobs"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  YouTube
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Employers"
              secondary={true}
            >
              <FooterLink
                analytics="toolbar:register+for+free"
                href="https://talent.seek.com.au/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Register for free
              </FooterLink>
              <FooterLink
                analytics="toolbar:post+a+job+ad"
                href="https://talent.seek.com.au/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Post a job ad
              </FooterLink>
              <FooterLink
                analytics="toolbar:products+prices"
                href="https://talent.seek.com.au/Support/OurProducts"
                linkRenderer={[Function]}
                secondary={false}
              >
                Products & prices
              </FooterLink>
              <FooterLink
                analytics="toolbar:customer+service"
                href="https://talent.seek.com.au/ContactUs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Customer service
              </FooterLink>
              <FooterLink
                analytics="toolbar:insight+blog"
                href="http://insightsresources.seek.com.au"
                linkRenderer={[Function]}
                secondary={false}
              >
                Insights & Resources
              </FooterLink>
            </FooterNav>
          </div>
          <nav
            className="copyright"
          >
            <ScreenReaderOnly>
              <h1>
                Additional Links
              </h1>
            </ScreenReaderOnly>
            <a
              className="copyrightLink"
              data-analytics="toolbar:privacy"
              href="/privacy"
            >
              Privacy
            </a>
            <a
              className="copyrightLink"
              data-analytics="toolbar:terms+conditions"
              href="/terms"
            >
              Terms & Conditions
            </a>
            <a
              className="copyrightLink"
              data-analytics="toolbar:seek+safely"
              href="/safe-job-searching"
            >
              SEEK safely
            </a>
            <a
              className="copyrightLink secondaryLink"
              data-analytics="toolbar:site+map"
              href="/sitemap"
            >
              Site Map
            </a>
            <p
              className="copyrightMessage"
            >
              © SEEK. All rights reserved.
            </p>
          </nav>
        </div>
      </section>
    </PageBlock>
  </Hidden>
</footer>
`;

exports[`Footer: should render with locale of NZ 1`] = `
<footer
  aria-labelledby="FooterHeading"
  className="root"
  role="contentinfo"
>
  <Hidden
    className=""
    desktop={false}
    mobile={false}
    print={true}
    screen={false}
  >
    <PageBlock>
      <section>
        <ScreenReaderOnly>
          <h1
            id="FooterHeading"
          >
            Footer
          </h1>
        </ScreenReaderOnly>
        <div
          className="content"
        >
          <div
            className="columns"
          >
            <FooterNav
              label="Tools"
              secondary={false}
            >
              <FooterLink
                analytics="toolbar:jobs"
                href="/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Job search
              </FooterLink>
              <FooterLink
                analytics="toolbar:profile"
                href="/profile/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Profile
              </FooterLink>
              <FooterLink
                analytics="toolbar:my+activity"
                href="/myactivity#favourite"
                linkRenderer={[Function]}
                secondary={false}
              >
                Saved searches
              </FooterLink>
              <FooterLink
                analytics="toolbar:favourite"
                href="/my-activity/saved-jobs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Saved jobs
              </FooterLink>
              <FooterLink
                analytics="toolbar:applied"
                href="/my-activity/applied-jobs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Applied jobs
              </FooterLink>
              <FooterLink
                analytics="toolbar:advice+tips"
                href="/career-advice/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Advice & tips
              </FooterLink>
              <ToggleContainer
                label="SEEK sites"
                name="PartnerSitesToggle"
                secondary={false}
              >
                <FooterLink
                  analytics="toolbar:post+a+job+ad"
                  href="https://talent.seek.co.nz/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Employer site
                </FooterLink>
                <FooterLink
                  analytics="toolbar:au+homepage"
                  href="https://www.seek.com.au/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  SEEK AU
                </FooterLink>
                <FooterLink
                  analytics="toolbar:courses"
                  href="https://www.seeklearning.co.nz/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Courses
                </FooterLink>
                <FooterLink
                  analytics="toolbar:business+for+sale"
                  href="https://www.seekbusiness.co.nz/?tracking=sk:main:nz:nav:bus"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Business for sale
                </FooterLink>
                <FooterLink
                  analytics="toolbar:volunteering"
                  href="https://www.volunteer.co.nz/?tracking=SKMNZ:main+header"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Volunteering
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Company"
              secondary={true}
            >
              <FooterLink
                analytics="toolbar:about+seek"
                href="/investor/about-us"
                linkRenderer={[Function]}
                secondary={true}
              >
                About SEEK
              </FooterLink>
              <FooterLink
                analytics="toolbar:media"
                href="/investor/news-media"
                linkRenderer={[Function]}
                secondary={true}
              >
                Media
              </FooterLink>
              <FooterLink
                analytics="toolbar:investor+centre"
                href="https://ir.seek.com.au"
                linkRenderer={[Function]}
                secondary={true}
              >
                Investor centre
              </FooterLink>
              <ToggleContainer
                label="International partners"
                name="InternationalPartnersToggle"
                secondary={true}
              >
                <FooterLink
                  analytics="toolbar:babajob"
                  href="http://www.babajob.com"
                  linkRenderer={[Function]}
                  partner="India"
                  secondary={false}
                >
                  BabaJob
                </FooterLink>
                <FooterLink
                  analytics="toolbar:bdjobs"
                  href="http://www.bdjobs.com"
                  linkRenderer={[Function]}
                  partner="Bangladesh"
                  secondary={false}
                >
                  Bdjobs
                </FooterLink>
                <FooterLink
                  analytics="toolbar:brighter+monday"
                  href="https://www.brightermonday.com"
                  linkRenderer={[Function]}
                  partner="E Africa"
                  secondary={false}
                >
                  Brighter Monday
                </FooterLink>
                <FooterLink
                  analytics="toolbar:catho"
                  href="http://www.catho.com.br"
                  linkRenderer={[Function]}
                  partner="Brazil"
                  secondary={false}
                >
                  Catho
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobberman"
                  href="http://www.jobberman.com"
                  linkRenderer={[Function]}
                  partner="W Africa"
                  secondary={false}
                >
                  Jobberman
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobsdb"
                  href="http://www.jobsdb.com"
                  linkRenderer={[Function]}
                  partner="SE Asia"
                  secondary={false}
                >
                  jobsDB
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jobstreet"
                  href="http://www.jobstreet.com"
                  linkRenderer={[Function]}
                  partner="SE Asia"
                  secondary={false}
                >
                  JobStreet
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jora+int"
                  href="https://www.jora.com"
                  linkRenderer={[Function]}
                  partner="Worldwide"
                  secondary={false}
                >
                  Jora
                </FooterLink>
                <FooterLink
                  analytics="toolbar:manager"
                  href="http://www.manager.com.br"
                  linkRenderer={[Function]}
                  partner="Brazil"
                  secondary={false}
                >
                  Manager
                </FooterLink>
                <FooterLink
                  analytics="toolbar:occ+mundial"
                  href="https://www.occ.com.mx"
                  linkRenderer={[Function]}
                  partner="Mexico"
                  secondary={false}
                >
                  OCC Mundial
                </FooterLink>
                <FooterLink
                  analytics="toolbar:workana"
                  href="https://www.workana.com"
                  linkRenderer={[Function]}
                  partner="Latin America"
                  secondary={false}
                >
                  Workana
                </FooterLink>
                <FooterLink
                  analytics="toolbar:zhaopin"
                  href="http://www.zhaopin.com"
                  linkRenderer={[Function]}
                  partner="China"
                  secondary={false}
                >
                  Zhaopin
                </FooterLink>
                <FooterLink
                  analytics="toolbar:jora+au"
                  href="https://au.jora.com"
                  linkRenderer={[Function]}
                  partner="Australia"
                  secondary={false}
                >
                  Jora
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Connect"
              secondary={false}
            >
              <FooterLink
                analytics="toolbar:contact+us"
                href="/contact-us"
                linkRenderer={[Function]}
                secondary={false}
              >
                Contact us / FAQs
              </FooterLink>
              <FooterLink
                analytics="toolbar:work+for+seek"
                href="/work-for-seek"
                linkRenderer={[Function]}
                secondary={true}
              >
                Work for SEEK
              </FooterLink>
              <FooterLink
                analytics="toolbar:product+blog"
                href="http://productblog.seek.com.au"
                linkRenderer={[Function]}
                secondary={true}
              >
                Product & Tech Blog
              </FooterLink>
              <ToggleContainer
                label="Social"
                name="SocialToggle"
                secondary={false}
              >
                <FooterLink
                  analytics="toolbar:facebook"
                  href="https://www.facebook.com/seeknz/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Facebook
                </FooterLink>
                <FooterLink
                  analytics="toolbar:instagram"
                  href="https://www.instagram.com/seeknz/"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Instagram
                </FooterLink>
                <FooterLink
                  analytics="toolbar:twitter"
                  href="https://twitter.com/seekjobsnz"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  Twitter
                </FooterLink>
                <FooterLink
                  analytics="toolbar:youtube"
                  href="https://www.youtube.com/user/SEEKJobs"
                  linkRenderer={[Function]}
                  secondary={false}
                >
                  YouTube
                </FooterLink>
              </ToggleContainer>
            </FooterNav>
            <FooterNav
              label="Employers"
              secondary={true}
            >
              <FooterLink
                analytics="toolbar:register+for+free"
                href="https://talent.seek.co.nz/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Register for free
              </FooterLink>
              <FooterLink
                analytics="toolbar:post+a+job+ad"
                href="https://talent.seek.co.nz/"
                linkRenderer={[Function]}
                secondary={false}
              >
                Post a job ad
              </FooterLink>
              <FooterLink
                analytics="toolbar:products+prices"
                href="https://talent.seek.co.nz/Support/OurProducts"
                linkRenderer={[Function]}
                secondary={false}
              >
                Products & prices
              </FooterLink>
              <FooterLink
                analytics="toolbar:customer+service"
                href="https://talent.seek.co.nz/ContactUs"
                linkRenderer={[Function]}
                secondary={false}
              >
                Customer service
              </FooterLink>
              <FooterLink
                analytics="toolbar:insight+blog"
                href="http://insightsresources.seek.co.nz"
                linkRenderer={[Function]}
                secondary={false}
              >
                Insights & Resources
              </FooterLink>
            </FooterNav>
          </div>
          <nav
            className="copyright"
          >
            <ScreenReaderOnly>
              <h1>
                Additional Links
              </h1>
            </ScreenReaderOnly>
            <a
              className="copyrightLink"
              data-analytics="toolbar:privacy"
              href="/privacy"
            >
              Privacy
            </a>
            <a
              className="copyrightLink"
              data-analytics="toolbar:terms+conditions"
              href="/terms"
            >
              Terms & Conditions
            </a>
            <a
              className="copyrightLink"
              data-analytics="toolbar:seek+safely"
              href="/safe-job-searching"
            >
              SEEK safely
            </a>
            <a
              className="copyrightLink secondaryLink"
              data-analytics="toolbar:site+map"
              href="/sitemap"
            >
              Site Map
            </a>
            <p
              className="copyrightMessage"
            >
              © SEEK. All rights reserved.
            </p>
          </nav>
        </div>
      </section>
    </PageBlock>
  </Hidden>
</footer>
`;
